### DISCUSSION TITLE
Github Support Not answer my ticket

### DISCUSSION THREAD

**Discussion by ghost** [community/community#47016] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

why github support not answer my ticket, what did i????

**Comment by SilasRodrigues19** [comment]
Hey @UmutKDev 

They really take a long time to answer, due to too many tickets

**Comment by queenofcorgis** [comment, marked as answer]
Hi there! Support answers tickets in the order tickets were received based on their own resources. I can't give you a time estimate, but rest assured our Support team will answer. I'm going to lock this post as the community is unable to assist here - we appreciate your patience ❤️ .
